src/main/java/com/oviva/konnektor/watchdog/Main.java

```diff
@@ -133,13 +133,19 @@ private void updateCardCheckGauges(
 
     var checker = new CardCheckGauges(config, konnektorFactory);
     gauges.register(checker, true);
-    logger.atInfo().log(
-        "registered gauges for all cards in konnektor {}", config.konnektorUri().toString());
+    logger
+        .atInfo()
+        .addKeyValue("konnektor", config.konnektorUri().toString())
```

Comment:
maybe just extract `config.konnektorUri().toString()` to a variable since it is used 4 times